src/components/vice/admin/index.js

```diff
@@ -4,12 +4,9 @@ import { useTranslation } from "i18n";
 import { useQuery, useMutation, queryCache } from "react-query";
 
 import { makeStyles } from "@material-ui/styles";
+import WrappedErrorHandler from "../../utils/error/WrappedErrorHandler";
```

Comment:
Not an issue at all but reminder that we can use absolute imports - `components/utils/error/WrappedErrorHandler`.